app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(31, 13)) == 31
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(21, 3) == [7, 7, 7]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(17, 1) == [17]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert split_integer(47, 11) == [4, 4, 4, 4, 4, 4, 4, 4, 5, 5, 5]
```

Comment:
You can use "sorted" here instead of hard-coding the values

Reply:
I don`t get your idea, can you please elaborate?

Reply:
@ada-krav 
The idea was to use `assert split_integer(47, 11) == sorted(split_integer(47,11))` instead `split_integer(47, 11) == [4, 4, 4, 4, 4, 4, 4, 4, 5, 5, 5]`
But after morning lecture I am not sure that is right case and, maybe, just write `[4, 4, 4, 4, 4, 4, 4, 4, 5, 5, 5]` like you did is better case then `sorted(split_integer(47,11))`.
I suppose when a mentor will check he shares his opinion about this)))

Reply:
@ZhAlexR u can make this in both ways